docs/projects/README.md

```diff
@@ -0,0 +1,469 @@
+
+- Links to existing issues or draft PRs related to the project
+- References to similar functionality in other projects, or 3rd party
+  documentation for a relevant integration
+- Links to past chat threads related to the project
+
+In conclusion, the deliverable for this stage is a new project thread which
+expresses our intent to work on a feature and why. The project thread should
+also be added to our GitHub Project Board, which can be done using the
+"Projects" section of the sidebar when viewing an issue in the GitHub UI. To
+learn more about our use of project boards please see the
+[Managing and working with projects](#managing-and-working-with-projects)
+section of this document.
+
+### Kickoff (`status: In Kickoff`)
+
+In the kickoff stage, a **project proposal** is written and shared with the team
+by the project lead. The primary goals of this document are that of discovery;
+uncovering assumptions we've made about the project's goals and implementation.
+Depending on the project, this discovery process may weigh more heavily towards
+infrastructural and other technical considerations or towards team processes and
+execution strategy.
+
+#### Required content
+
+Every plan should contain the following information:
+
+1. Which of our year goals does this project advance?
+2. How will we measure the success of this project? Share any new analytics
+   tracking or metrics we will need to evaluate the success of the project.
+3. Who will work on the project? Consider any sponsored developers needed, along
+   with any external contributors or collaborators who need to be involved.
+4. Are there any infrastructural changes or costs associated with this project?
+5. Is there any marketing outreach that should be coordinated with this project?
+   How is the project presented and made visible to our users?
+6. Will any outside stakeholders or communities need to approve parts of the
+   project, or confirm their interest?
+7. A list of the requested technical implementation plans for the projects. Some
+   projects will require multiple implementation plans (one for each the
+   catalogue, API, and frontend, for example), and capturing that now will help
+   understand the scope of the planning and implementation work that will arise
+   down the road.
+
+Leads are not expected to know the answers to all of these questions. Rather,
+leads should **ask questions** directly in the document, tagging relevant
+contributors who can bring needed expertise and guidance. The goal is to lead
+the process of requirement and assumption discovery.
+
+> **Note**
+>
+> **On Analytics**: Leads must endeavour to describe metrics and analysis that
+> should be completed before the changes for the project's ultimate goal are
+> implemented. Whenever possible or relevant, projects should have success
+> criteria that compare the state of relevant metrics after the project's
+> completion to the same metrics beforehand.
+
+#### Tone and audience
```

Comment:
Thank you for providing this section! It's really helpful to know the tone /audience before writing something!